src/main/java/com/oltpbenchmark/benchmarks/tpch/TPCHLoader.java

```diff
@@ -169,38 +146,16 @@ private boolean loadCopy(Connection conn) {
         final CountDownLatch supplierLatch = new CountDownLatch(1);
         final CountDownLatch partsSuppLatch = new CountDownLatch(1);
 
-        threads.add(new LoaderThread(this.benchmark) {
-            @Override
-            public void load(Connection conn) throws SQLException {
-                if (loadCopy(conn)) {
-                    copySuccess[0] = true;
-                }
-            }
-
-            @Override
-            public void afterLoad() {
-                copyLatch.countDown();
-            }
-        });
+        final double scaleFactor = this.workConf.getScaleFactor();
 
         threads.add(new LoaderThread(this.benchmark) {
             @Override
             public void load(Connection conn) throws SQLException {
-                if (copySuccess[0]) {
-                    LOG.info("Skipping LOAD of Region because COPY has been done.");
-                    return;
-                }
                 try (PreparedStatement statement = conn.prepareStatement("INSERT INTO region " + " (r_regionkey, r_name, r_comment) " + "VALUES (?, ?, ?)")) {
-                    loadTable(conn, statement, "Region", regionTypes);
-                }
-            }
+                    List<Iterable<List<Object>>> regionGenerators = new ArrayList<>();
+                    regionGenerators.add(new RegionGenerator());
 
-            @Override
-            public void beforeLoad() {
-                try {
-                    copyLatch.await();
-                } catch (InterruptedException e) {
-                    throw new RuntimeException(e);
+                    genTable(conn, statement, regionGenerators, regionTypes, "Region");
```

Comment:
Don't use hardcoded strings for table names. Define them in `TPCHConstants`.

https://github.com/cmu-db/benchbase/blob/main/src/main/java/com/oltpbenchmark/benchmarks/tatp/TATPConstants.java#L39-L52